bokeh/command/__main__.py

```diff
@@ -0,0 +1,8 @@
+'''
+
+'''
+import sys
+
+from .bootstrap import main
+
+main(sys.argv)
```

Comment:
`python -m` just loads a module (specified by `-m`) before doing anything else, but in the case of a package with a `__main__.py` it executes that. So this currently allows you to do 
```
python -m bokeh.command server <stuff>
```
which was my only intention. I'm +0 on making the `bokeh` module as a whole executable, but if others feel strongly, we could add a `__main__` there too.